tests/affiliatedcertification/tests/affiliated_certification_operator.go

```diff
@@ -95,41 +100,46 @@ var _ = Describe("Affiliated-certification operator certification,", func() {
 			Expect(err).ToNot(HaveOccurred(), "Error validating test reports")
 		})
 
-	// 46582
-	It("one operator to test, operator belongs to certified-operators organization in Red Hat catalog"+
-		" and its version is certified", func() {
-		By("Deploy operators to test")
-
-		operatorGroup := utils.DefineOperatorGroup("affiliatedcert-test-operator-group",
-			affiliatedcertparameters.TestCertificationNameSpace,
-			[]string{affiliatedcertparameters.TestCertificationNameSpace})
+	// 46697
+	It("two operators to test, one belongs to certified-operators organization and its version is certified,"+
+		" one does not belong to certified-operators organization in Red Hat catalog [skip]", func() {
+		By("Deploy operator to test")
 
 		certifiedOperatorPostgresSubscription := utils.DefineSubscription("crunchy-postgres-operator-subscription",
 			affiliatedcertparameters.TestCertificationNameSpace, "v5", "crunchy-postgres-operator",
 			affiliatedcertparameters.CertifiedOperatorGroup, affiliatedcertparameters.OperatorSourceNamespace)
 
 		err := affiliatedcerthelper.DeployOperator(affiliatedcertparameters.TestCertificationNameSpace,
-			operatorGroup,
 			certifiedOperatorPostgresSubscription)
-		Expect(err).ToNot(HaveOccurred(), "Error deploying operator")
+		Expect(err).ToNot(HaveOccurred(), "Error deploying operator "+
+			affiliatedcertparameters.CertifiedOperatorPrefixPostgres)
 
 		By("Confirm that operator is installed and ready")
 
 		Eventually(func() bool {
 			err = affiliatedcerthelper.IsOperatorInstalled(affiliatedcertparameters.TestCertificationNameSpace, "pgo")
 
 			return err == nil
-		}, 5*time.Minute, 5*time.Second).Should(Equal(true), "Operator is not ready")
+		}, 5*time.Minute, 5*time.Second).Should(Equal(true), "Operator "+
```

Comment:
I would suggest using vars that will be the eventually time args, it would be easier for you to debug/ control it 
in future debugging when needed. 
other than that, LGTM.